### PR TITLE
Update electron version

### PR DESCRIPTION
**Pull request by unellma**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
     "css-loader": "^6.7.1",
     "dateformat": "^5.0.3",
     "dotenv": "^16.0.1",
-    "electron": "^v27.1.2",
+    "electron": "^v28.1.0",
```
Comment (appketo) [inline package.json:83]: Why not update to latest version [v28.1.4](https://releases.electronjs.org/release/v28.1.4)

**Comment by unellma** [comment]
[UnelmaSupport](https://unelmasupport.com)

# Hello!

Thanks for getting in touch! This is an automatic email just to let you know that we've received your request and your ticket reference is #306.

One of our support agents will get back to you shortly. Please do not submit multiple tickets for the same request, as this will not result in a faster response time.

View Help Center: https://unelmasupport.com/hc

Regards,
UnelmaSupport

If you’re having trouble clicking the "View Help Center" button, copy and paste the URL below into your web browser: [https://unelmasupport.com/hc](https://unelmasupport.com/hc)

© 2024 UnelmaSupport. All rights reserved.

File: package.json (hunk 1 above)
Comment (unellma) [reply package.json:83]: ok, let me check it

**Review by appketo — APPROVED** [review]

**Comment by unellma** [comment]
[UnelmaSupport](https://unelmasupport.com)

# Hello!

Thanks for getting in touch! This is an automatic email just to let you know that we've received your request and your ticket reference is #307.

One of our support agents will get back to you shortly. Please do not submit multiple tickets for the same request, as this will not result in a faster response time.

View Help Center: https://unelmasupport.com/hc

Regards,
UnelmaSupport

If you’re having trouble clicking the "View Help Center" button, copy and paste the URL below into your web browser: [https://unelmasupport.com/hc](https://unelmasupport.com/hc)

© 2024 UnelmaSupport. All rights reserved.

**Pull request merged**